### ISSUE TITLE
A11y_Model Builder_Add Data_Input_Name Role Value : Name is defined incorrectly for "Time Series Column" tooltip.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2187]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Client; #SH-ModelBuilder-Win32-Jun2022; #Win32; #WCAG4.1.2; #Name, Role, Value;

### Environment Details:
Application Name: Model Builder
Windows Version: Windows 11 22H2 OS Builder 25145.1011
Visual Studio: Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview Version 17.3.0 Preview 2.0

### Repro Steps:

1. Install Visual studio and open Visual studio
2. Navigate to C# project name and right click on it
3. Navigate to "Add" button and press enter
4. Navigate to "Machine learning model" and press enter
5. "Add new item" popup will open
6. Navigate to "Add" button and press enter
7. "Select scenario" popup will open and select "Forecasting" scenario
8. "Select training environment" page will open then select "Next step"
9. "Add Data" page will appear tab till Input section.
10. Tab till “Time series column” info/tooltip.
11. Check whether the name is defined correctly for "Time Series Column" info/tooltip or not.

### Actual Result:
Name is defined incorrectly for "Time Series Column" info/tooltip as "Column To Predict".

### Expected Result:
Name should be defined correctly for "Time Series Column" info/tooltip.

![A11y_Model Builder_Add Data_Input_Name Role Value](https://user-images.githubusercontent.com/91586101/177528326-6051b177-ce5e-40c0-aa6b-9f2318a56588.png)

**Comment by beccamc** [comment]
@manaswita-chougule  Please verify this on build 16.13.10.2241902 available from the Marketplace https://marketplace.visualstudio.com/items?itemName=MLNET.ModelBuilder2022

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment:
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.4.0 Preview 1.0

The issue is fixed, hence closing the bug. Please find below attachment for reference.

![#2187_fixed_Time series col name](https://user-images.githubusercontent.com/91586101/187127358-38e4c58c-15b0-4588-bf24-7974a097824c.png)
